GMAO_pFIO/tests/CMakeLists.txt

```diff
@@ -32,13 +32,13 @@ set (SRCS
 add_library (pfio_extras
   pFIO_Initialize.F90
   )
-target_link_libraries (pfio_extras PUBLIC MAPL_pFUnit)
+target_link_libraries (pfio_extras PUBLIC MAPL_pFUnit MAPL_Shared)
 
 
 add_pfunit_ctest(pFIO_tests 
                 TEST_SOURCES ${TEST_SRCS}
                 OTHER_SOURCES ${SRCS}
-                LINK_LIBRARIES GMAO_pFIO pfio_extras MAPL_pFUnit
+                LINK_LIBRARIES GMAO_pFIO pfio_extras MAPL_pFUnit MAPL_Shared
```

Comment:
Just FYI:  I suspect this line only needs the 1st two items.  The others are implied through transitivity.

Reply:
The last two are removed